src/XacroParser.js

```diff
@@ -535,6 +540,14 @@ export class XacroParser {
 
                 return rospackCommands(stem, ...args);
 
+            } else if (!(stem in rospackCommands) && stem === 'arg') {
+
+                const arg = args[0];
+                if (arg != null && this.arguments[arg] != null) {
```

Comment:
You mean like `!== undefined`? What should we do if someone sets an argument value to null? It doesn't really make sense to substitute null into an expression.

Reply:
I've been using loose equality for null/undefined because `null==undefined` but `null!=` anything else, and vice versa: https://dorey.github.io/JavaScript-Equality-Table/

Reply:
Yes sorry it wasn't immediately clear what was important. I've grown used to just using strict equality for everything.

Reply:
Yep me too, with null & undefined being the only exception. It seems you do have an eslint rule enforcing strict equality, but it has a config option specifically for null which allows this: https://eslint.org/docs/rules/eqeqeq